### ISSUE TITLE
[Feature requests] set/overwrite fake timestamps (fixed constant velocity; or time difference divided by fixed number of points) 

### ISSUE THREAD

**Issue by Wikinaut** [remisalmon/gpx_interpolate#19]
For setting track data after words for uploading to Mapillary, I the script needs two further options regardiing the set up of timestamps.

When timestamps are missing, creating of reasonable timestamps is needed for some applications like Mapillary.

Further input parameters, which are needed, are then:

* start timestamp (optional plus end timestamp)
* constant velocity (meter/second) or parameters like "assume 2 Meter difference between each track point and 1 second difference"

tl;dr:

anything which allow to assign reasonable timestamps to the interpolated track.

**Comment by Wikinaut** [comment]
fixed/implemented in my version (a fork) https://github.com/Wikinaut/gpx-interpolate
